doc/classes/AnimationNodeStateMachineTransition.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AnimationNodeStateMachineTransition" inherits="Resource" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		A resource to connect each node to make a path for [AnimationNodeStateMachine].
```

Comment:
Could be good to add the `<description>` too while at it, if you have an idea for what to write.

Reply:
I wrote a little bit about it.